.github/workflows/tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
```

Comment:
Let's try a supported Node LTS. 14 would be best if it works

Reply:
Ok, bumped to 14